Vagrantfile

```diff
@@ -20,11 +20,14 @@ Vagrant.configure(2) do |config|
     config.vm.provision "shell", inline: <<-EOF
         set -eu
 
-        sudo dnf install -y cockpit
+        sudo dnf install -y cockpit podman
 
         printf "[WebService]\nAllowUnencrypted=true\n" > /etc/cockpit/cockpit.conf
 
         systemctl enable cockpit.socket
         systemctl start cockpit.socket
+
+        systemctl enable podman
+        systemctl start podman
```

Comment:
Shouldn't that be `io.podman.socket`? The .service unit shouldn't have an `[Install]` section in the first place, that's a bug in podman. Do you know what the difference is between the two sockets? Is podman.socket the new-style REST API (which current master doesn't support yet, @marusak is working on it), and io.podman.socket the varlink API?

Reply:
Indeed, `podman.{socket,service}` is the new REST API that is not yet supported and `io.podman.{socket,service}` is the varlink one, that we currently support.